c_src/workitems.cc

```diff
@@ -691,20 +874,20 @@ char* EncodeVarint64(char* dst, uint64_t v) {
  */
 work_result RangeScanTask::operator()()
```

Comment:
Please see second catch block.  github will not let me comment on line 1063 of new code.  ErlUtil::formatBinary() can send a throw.  What happens to a throw that occurs during a catch?  I don't know.